### PR TITLE
Update GCC to 13.1.0 for EL9

### PR DESCRIPTION
**Pull request by iarspider**

### CODE REVIEW

**Comment by iarspider** [comment]
hold

File: bazel.spec [hunk 1]
```diff
@@ -17,20 +17,24 @@ BuildRequires: java-env python3 python-python3
...
 Patch0: bazel-3.7.0-patches
-Patch1: bazel-3.7.2-gcc11
+Patch1: bazel-absl
 
-%prep
-
-%define __unzip unzip -d bazel-%{realversion}
+%if "%{?cms_cxx:set}" != "set"
+%define cms_cxx c++
+%endif
```
Comment (smuzaffar) [inline bazel.spec]: do we need this change?

File: bazel.spec [hunk 2]
```diff
@@ -17,20 +17,24 @@ BuildRequires: java-env python3 python-python3
...
+%setup -q -c -n bazel-%{realversion}
 
 %patch0 -p1
 %patch1 -p1
 
 %build
 
 export EXTRA_BAZEL_ARGS="--host_javabase=@local_jdk//:jdk --jobs %{compiling_processes}"
+#export CXX="$(which %{cms_cxx}) -std=c++17"
+#export CC="$(which gcc) -std=c++17"
```
Comment (smuzaffar) [inline bazel.spec]: and this one?

**Comment by smuzaffar** [comment]
merging this for IB/CMSSW_13_2_X/g13 branch. Tests https://cmssdt.cern.ch/jenkins/job/jenkins-test-bootstrap/241/ works for building bootstrap.

**Pull request merged**